Packages/com.unity.inputsystem/InputSystem/Devices/Keyboard.cs

```diff
@@ -842,9 +875,86 @@ public enum Key
         /// </summary>
         OEM5,
 
-        ////FIXME: This should never have been a Key but rather just an extra button or state on keyboard
-        // Not exactly a key, but binary data sent by the Keyboard to say if IME is being used.
-        IMESelected
+        /// <summary>
+        /// Don't use this. This is a dummy key that is only used internally to represent the IME selected state.
+        /// Will be removed in the future.
+        /// </summary>
+        [Obsolete("Don't use this. This is a dummy key that is only used internally to represent the IME selected state. Will be removed in the future.", true)]
```

Comment:
Older Input Systems would still rely on this bit

Reply:
does the old input system use this?